### PR TITLE
Admin session management and Rspec unit test cases

### PR DESCRIPTION
**Pull request by sourabh-sawant-v2**
1. New index page for members controller is added in community with links to members page to list all members and link to add new member page.
2. Added: Login controller is added to generate login session token.
3. Added: Admin authentication and session management using session ID.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
+  root 'index_pages#index'
```
Comment (shileshv2) [inline config/routes.rb]: Why root is changed from existing `pages` to `index_pages`

File: config/database.yml [hunk 2]
```diff
@@ -15,21 +15,17 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host:
```
Comment (shileshv2) [inline config/database.yml:18]: Add `database.yml` to gitignore file

**Review by shileshv2 — CHANGES_REQUESTED** [review]
No need to create `index_pages_controller` when you have existing `pages`  controller

File: config/routes.rb (hunk 1 above)
Comment (sourabh-sawant-v2) [reply config/routes.rb]: changed root path to pages route

File: config/database.yml (hunk 2 above)
Comment (sourabh-sawant-v2) [reply config/database.yml:18]: Added database.yml to gitignore file

File: app/views/family_members/index.html.erb [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <p style="color: green"><%= notice %></p>
 
+<button><%= link_to "Home", pages_path, class: "btn btn-primary" %></button>
```
Comment (shilesh) [inline app/views/family_members/index.html.erb]: move repeating code to a reusable file, like partial

**Review by shilesh — CHANGES_REQUESTED** [review]
Reuse repeating code
Cookies required if session is stored in session table ?

File: app/views/family_members/index.html.erb (hunk 3 above)
Comment (sourabh-sawant-v2) [reply app/views/family_members/index.html.erb]: repeating code is moved to partial file.

**Pull request merged**